py/server/deephaven/stream/kafka/consumer.py

```diff
@@ -280,6 +283,55 @@ def _consume(
     except Exception as e:
         raise DHError(e, "failed to consume a Kafka stream.") from e
 
+def protobuf_spec(
+        schema: str,
+        schema_version: Optional[int] = None,
+        schema_message_name: Optional[str] = None,
+        include: Optional[List[str]] = None,
+) -> KeyValueSpec:
+    """The kafka protobuf specs. This will fetch the protobuf descriptor for the schema subject from the schema registry
+    using version schema_version and create protobuf message parsing functions according to parsing options. These
+    functions will be adapted to handle schema changes.
+
+    For purposes of reproducibility across restarts where schema changes may occur, it is advisable for callers
+    to set a specific schema_version. This will ensure the resulting table definition will not change across restarts.
+    This gives the caller an explicit opportunity to update any downstream consumers when updating schema_version if
+    necessary.
+
+    Args:
+        schema (str): the schema subject name
+        schema_version (Optional[int]): the schema version, or None for latest, default is None. For purposes of
+            reproducibility across restarts where schema changes may occur, it is advisable for callers to set this.
+            This will ensure the resulting table definition will not change across restarts. This gives the caller an
+            explicit opportunity to update any downstream consumers when updating schema_version if necessary.
+        schema_message_name (Optional[str]): the schema message name, whose descriptor will be used to inform the
+            resulting table definition, or None to use the first message definition in the schema. The default is None.
+            It is advisable for callers to explicitly set this.
```

Comment:
This will not be clear to an end user.

Reply:
What exact part won't be clear? Happy to take suggestions on rewording to make clearer.

Reply:
I have a mediocre knowledge of Kafka, so this block comes across as insider jargon.  Providing some definition, context, or whatever would help to clarify the jargon would be great.